yarn-project/aztec.js/src/contract/sent_tx.ts

```diff
@@ -80,6 +80,14 @@ export class SentTx {
     return receipt.status === TxStatus.MINED;
   }
 
+  /**
+   * Gets unencrypted logs emitted by this tx.
+   * @returns The requested logs.
+   */
+  public async getUnencryptedLogs(): Promise<ExtendedUnencryptedL2Log[]> {
+    return this.pxe.getUnencryptedLogs({ txHash: await this.getTxHash() });
+  }
```

Comment:
Love this addition! But should we `wait()` the tx to be mined before calling the pxe for its logs?

Reply:
Good idea. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/2600/commits/64fc40a9039bf4ab0118830fd95dc9b02d8539cb